source/appModules/winword.py

```diff
@@ -23,6 +49,10 @@ def chooseNVDAObjectOverlayClasses(self, obj, clsList):
 
 
 class WinwordWordDocument(WordDocument):
+	def _get_description(self) -> str:
+		curView = self.WinwordWindowObject.view.Type
```

Comment:
What does this do when the Word object model isn't available (for instance when `config.isAppX`)?

Reply:
I'm a little surprised that there doesn't appear to be a UIA custom extension for this. I'll ask around internally

Reply:
I'm a little surprised that there doesn't appear to be a UIA custom extension for this. I'll ask around internally

CC @rperez030 for tracking.